Sources/Recycling/ListViews/AdManagement/Cells/UserAdManagementTransactionProcessCell.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  Copyright © 2020 FINN AS. All rights reserved.
+//
+
+public class UserAdManagementTransactionProcessCell: UITableViewCell {
+    // MARK: - Public
+
+    private(set) public var model: TransactionProcessViewModel?
```

Comment:
Does the `model` need to be public?

Reply:
Not right now, but that may change in the future. I can keep it private until it's actually needed 👍